rules_java_gapic/java_gapic.bzl

```diff
@@ -24,6 +24,7 @@ def _java_gapic_postprocess_srcjar_impl(ctx):
     output_samples = ctx.outputs.samples
     output_resource_name = ctx.outputs.resource_name
     formatter = ctx.executable.formatter
+    generator_version = "2.21.1-SNAPSHOT"  # {x-version-update:gapic-generator-java:current}
```

Comment:
Rather than declaring a new value, can you use the version defined in WORKSPACE file https://github.com/googleapis/googleapis/blob/06f7999f85dfe75fffaf30c5ca4d9aa6b58669f8/WORKSPACE#L273 ? Otherwise the two variables might point to different values.